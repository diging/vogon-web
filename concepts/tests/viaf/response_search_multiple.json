{
  "query": "einstein",
  "result": [
      {
          "term": "Einstein, Albert, 1879-1955",
          "displayForm": "Einstein, Albert, 1879-1955",
          "nametype": "personal",
          "lc": "n79022889",
          "dnb": "118529579",
          "selibr": "184709",
          "bav": "495_91752",
          "bnf": "11901607",
          "jpg": "500240971",
          "iccu": "cfiv035853",
          "bne": "xx834035",
          "egaxa": "vtls000837931",
          "nkc": "jn19990002019",
          "nla": "000036582360",
          "ptbnp": "26339",
          "viafid": "75121530",
          "score": "26530",
          "recordID": "75121530"
      },
      {
          "term": "Einstein, Alfred, 1880-1952",
          "displayForm": "Einstein, Alfred, 1880-1952",
          "nametype": "personal",
          "lc": "n50030286",
          "dnb": "116425725",
          "selibr": "184710",
          "bav": "495_73269",
          "bnf": "12409589",
          "bne": "xx1722499",
          "egaxa": "vtls000820561",
          "nkc": "ola2002161255",
          "nla": "000035059692",
          "ptbnp": "479052",
          "viafid": "61634867",
          "score": "7912",
          "recordID": "61634867"
      },
      {
          "term": "Einstein-Marić, Mileva, 1875-1948",
          "displayForm": "Einstein-Marić, Mileva, 1875-1948",
          "nametype": "personal",
          "lc": "n85163829",
          "dnb": "118688405",
          "selibr": "184711",
          "bnf": "12045459",
          "iccu": "ravv083778",
          "bne": "xx887199",
          "nkc": "jx20050826002",
          "viafid": "7408788",
          "score": "5495",
          "recordID": "7408788"
      },
      {
          "term": "Einstein-Sammlung",
          "displayForm": "Einstein-Sammlung",
          "nametype": "corporate",
          "lc": "n80126021",
          "dnb": "004813251",
          "selibr": "113594",
          "bav": "494_66254",
          "bnf": "11882844",
          "jpg": "500312234",
          "bne": "xx91316",
          "nkc": "ko2002102420",
          "nla": "000036098673",
          "viafid": "129675534",
          "score": "4893",
          "recordID": "129675534"
      },
      {
          "term": "Einstein, Carl, 1885-1940",
          "displayForm": "Einstein, Carl, 1885-1940",
          "nametype": "personal",
          "lc": "n80104380",
          "dnb": "118529587",
          "selibr": "296734",
          "bav": "495_352889",
          "bnf": "12038629",
          "jpg": "500248592",
          "nkc": "xx0058030",
          "nla": "000035799060",
          "viafid": "34473997",
          "score": "4610",
          "recordID": "34473997"
      },
      {
          "term": "Einstein, Arik",
          "displayForm": "Einstein, Arik",
          "nametype": "personal",
          "lc": "n83184374",
          "dnb": "1191035891",
          "bnf": "14174704",
          "viafid": "71807403",
          "score": "2432",
          "recordID": "71807403"
      },
      {
          "term": "Einstein, Lewis, 1877-1967",
          "displayForm": "Einstein, Lewis, 1877-1967",
          "nametype": "personal",
          "lc": "n50030287",
          "dnb": "101553234",
          "bav": "495_144382",
          "nkc": "mzk2014823241",
          "nla": "000035059714",
          "viafid": "52060042",
          "score": "2126",
          "recordID": "52060042"
      },
      {
          "term": "Einstein, Charles",
          "displayForm": "Einstein, Charles",
          "nametype": "personal",
          "lc": "n79041933",
          "dnb": "1062119568",
          "bnf": "11105204",
          "nla": "000035059707",
          "viafid": "48035085",
          "score": "1934",
          "recordID": "48035085"
      },
      {
          "term": "Einstein, Albert Lawrence",
          "displayForm": "Einstein, Albert Lawrence",
          "nametype": "personal",
          "lc": "n87921590",
          "dnb": "130586552",
          "bnf": "14030953",
          "bne": "xx1085471",
          "nkc": "osd2013750362",
          "viafid": "118902208",
          "score": "1928",
          "recordID": "118902208"
      },
      {
          "term": "Einstein, Hans Albert, 1904-1973",
          "displayForm": "Einstein, Hans Albert, 1904-1973",
          "nametype": "personal",
          "lc": "n85802437",
          "dnb": "143075845",
          "viafid": "67420636",
          "score": "1626",
          "recordID": "67420636"
      }
  ]
}