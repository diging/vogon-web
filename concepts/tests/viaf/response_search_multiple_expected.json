[
  {
    "local_identifier": "75121530",
    "concept_type": "viaf:personal",
    "description": "Einstein, Albert, 1879-1955",
    "name": "Einstein, Albert, 1879-1955",
    "identifier": "http://viaf.org/viaf/75121530"
  },
  {
    "local_identifier": "61634867",
    "concept_type": "viaf:personal",
    "description": "Einstein, Alfred, 1880-1952",
    "name": "Einstein, Alfred, 1880-1952",
    "identifier": "http://viaf.org/viaf/61634867"
  },
  {
    "local_identifier": "7408788",
    "concept_type": "viaf:personal",
    "description": "Einstein-Mari\u0107, Mileva, 1875-1948",
    "name": "Einstein-Mari\u0107, Mileva, 1875-1948",
    "identifier": "http://viaf.org/viaf/7408788"
  },
  {
    "local_identifier": "129675534",
    "concept_type": "viaf:corporate",
    "description": "Einstein-Sammlung",
    "name": "Einstein-Sammlung",
    "identifier": "http://viaf.org/viaf/129675534"
  },
  {
    "local_identifier": "34473997",
    "concept_type": "viaf:personal",
    "description": "Einstein, Carl, 1885-1940",
    "name": "Einstein, Carl, 1885-1940",
    "identifier": "http://viaf.org/viaf/34473997"
  },
  {
    "local_identifier": "71807403",
    "concept_type": "viaf:personal",
    "description": "Einstein, Arik",
    "name": "Einstein, Arik",
    "identifier": "http://viaf.org/viaf/71807403"
  },
  {
    "local_identifier": "52060042",
    "concept_type": "viaf:personal",
    "description": "Einstein, Lewis, 1877-1967",
    "name": "Einstein, Lewis, 1877-1967",
    "identifier": "http://viaf.org/viaf/52060042"
  },
  {
    "local_identifier": "48035085",
    "concept_type": "viaf:personal",
    "description": "Einstein, Charles",
    "name": "Einstein, Charles",
    "identifier": "http://viaf.org/viaf/48035085"
  },
  {
    "local_identifier": "118902208",
    "concept_type": "viaf:personal",
    "description": "Einstein, Albert Lawrence",
    "name": "Einstein, Albert Lawrence",
    "identifier": "http://viaf.org/viaf/118902208"
  },
  {
    "local_identifier": "67420636",
    "concept_type": "viaf:personal",
    "description": "Einstein, Hans Albert, 1904-1973",
    "name": "Einstein, Hans Albert, 1904-1973",
    "identifier": "http://viaf.org/viaf/67420636"
  }
]